src/client/eppo-client.ts

```diff
@@ -131,40 +132,32 @@ export default class EppoClient {
   private assignmentCache?: AssignmentCache;
   // whether to suppress any errors and return default values instead
   private isGracefulFailureMode = true;
-  private expectObfuscated: boolean;
-  private obfuscationMismatchWarningIssued = false;
-  private configObfuscatedCache?: boolean;
   private requestPoller?: IPoller;
   private readonly evaluator = new Evaluator();
   private configurationRequestor?: ConfigurationRequestor;
   private readonly overrideValidator = new OverrideValidator();
 
   constructor({
     eventDispatcher = new NoOpEventDispatcher(),
-    isObfuscated = false,
+    isObfuscated,
     flagConfigurationStore,
     banditVariationConfigurationStore,
     banditModelConfigurationStore,
     overrideStore,
     configurationRequestParameters,
-  }: {
-    // Dispatcher for arbitrary, application-level events (not to be confused with Eppo specific assignment
-    // or bandit events). These events are application-specific and captures by EppoClient#track API.
-    eventDispatcher?: EventDispatcher;
-    flagConfigurationStore: IConfigurationStore<Flag | ObfuscatedFlag>;
-    banditVariationConfigurationStore?: IConfigurationStore<BanditVariation[]>;
-    banditModelConfigurationStore?: IConfigurationStore<BanditParameters>;
-    overrideStore?: ISyncStore<Variation>;
-    configurationRequestParameters?: FlagConfigurationRequestParameters;
-    isObfuscated?: boolean;
-  }) {
+  }: EppoClientParameters) {
     this.eventDispatcher = eventDispatcher;
     this.flagConfigurationStore = flagConfigurationStore;
     this.banditVariationConfigurationStore = banditVariationConfigurationStore;
     this.banditModelConfigurationStore = banditModelConfigurationStore;
     this.overrideStore = overrideStore;
     this.configurationRequestParameters = configurationRequestParameters;
-    this.expectObfuscated = isObfuscated;
+
+    if (isObfuscated !== undefined) {
+      logger.info(
```

Comment:
minor: given this log is infrequent, I think `warn` is justified

Reply:
I still think that `warn` is OK here as otherwise nobody is going to see it 🙂 But I don't feel very strongly about it, so feel free to disagree and ignore

Reply:
hm. let's do `warn` and we'll revisit if a customer experiences issues with the volume of warnings (which we'll hopefully obviate but cutting a new major release)